ui/components/ui/url-icon/url-icon.stories.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
-import { text } from '@storybook/addon-knobs';
 import UrlIcon from './url-icon';
 
 export default {
   title: 'Components/UI/UrlIcon',
   id: __filename,
+  argType: {
+    name: { description: 'Icon Name', control: 'text' },
+    url: { description: 'Icon File URL', control: 'text' },
+    className: { control: 'text' },
+    fallbackClassName: { control: 'text' },
+  },
+  args: {
+    name: 'AST',
+    url: 'AST.png',
+    className: '',
+    fallbackClassName: '',
+  },
+};
+
+export const DefaultStory = (args) => {
+  return <UrlIcon name={args.name} url={args.url} />;
```

Comment:
The `DefaultStory` should accept all available `argTypes`. I usually just spread the `...args` but there has been some concern about when we convert to typescript so we can just declare them here too.

```suggestion
  return <UrlIcon name={args.name} url={args.url} className={args.className} fallbackClassName={fallbackClassName} />;
```